psydac/fem/vector.py

```diff
@@ -165,10 +165,10 @@ def __new__(cls, *spaces):
     def __init__( self, *spaces):
         """."""
 
-        if len(args) == 1:
+        if len(spaces) == 1:
             return
 
-        self._spaces = tuple(args)
+        self._spaces = tuple(spaces)
```

Comment:
By construction `spaces` is already a `tuple`, which is an immutable object. So there is no need to create a new tuple here, and the following suffices:
```python
self._spaces = spaces
```

Reply:
done